Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -485,6 +484,13 @@ theorem Memℒp.integrable_norm_rpow' [IsFiniteMeasure μ] {f : α → β} {p :
   · simp [h_top, integrable_const]
   exact hf.integrable_norm_rpow h_zero h_top
 
+lemma Memℒp.integrable_norm_pow {f : α → β} {p : ℕ} (hf : Memℒp f p μ) (hp : p ≠ 0) :
```

Comment:
These lemmas are new, right? Are you using them anywhere in this PR?

Reply:
Whoops, they were supposed to belong to another PR. Will move them out